includes/im_user_userheader_block.inc

```diff
@@ -48,18 +46,18 @@ function _im_user_userheader_bar() {
   $timestamp =strtotime($timestamp);
   $datedisplay = format_date($timestamp,'custom','l j F Y');
   $timedisplay = array('class'=>'im_user_portal_time');
-  $item_display[0] = $datedisplay;
-  $item_display[] = $timedisplay;
+  $item_display[0] = '';
+  //$item_display[] = $timedisplay;
   $count = 0;
-if(!empty ($user_fields->field_user_stores['und'][0]['value'])) {
-    $prefered_store = $user_fields->field_user_stores['und'][0]['value'];
+  if (!empty ($user_fields->field_user_stores['LANGUAGE_NONE'][0]['value'])) {

```

Comment:
`'LANGUAGE_NONE'` : this tells me you have attempted this work in a very rushed manner: remember we prefer **quality over speed** so in this case, you have no unit tests to save you, and you need to be more careful - we have a policy of **all changes require a test first** which helps a lot to save you from this kind of embarrassing situation